src/pages/blog/airship-update-july-2020.md

```diff
@@ -0,0 +1,138 @@
+[_blog posts_](https://www.airshipit.org/blog/) regularly, including the new Airship 2.0 Blog series. These blog posts
+introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail each component's evolution.
+The first six Airship 2.0 Blog posts are already available, and illustrate the design thoughts on the road to the alpha
+milestone:
+
+* [_Airship Blog Series 1 - Evolution Towards 2.0_](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+* [_Airship Blog Series 2 - An Educated Evolution_](
+  https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+* [_Airship Blog Series 3 - Airship 2.0 Architecture High Level_](
+  https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+* [_Airship Blog Series 4 - Shipyard - an Evolution of the Front Door_](
+  https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+* [_Airship Blog Series 5 - Drydock and Its Relationship to Cluster API_](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+* [_Airship Blog Series 6 - Armada Growing Pains_](
+  https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+<br>
+
+Stay tuned for more exciting news on the Airship blog series as Airship 2.0 progresses through the beta milestone and
+beyond!
+
+<br>
+
+## **AIRSHIP PROJECT UPDATE**
+
+Each quarter the OpenStack Foundation hosts a community meeting where the latest news about OpenStack and top-level
+projects can be shared. On 25 June, Airship presented a project update alongside OpenStack, Kata Containers, StarlingX,
+Zuul, and pilot project OpenInfra Labs.
+
+Catch up on the community meeting with these resources:
+* [_Recording_](https://zoom.us/rec/share/vJF_FqPgxGJJQ9bntR7vaqM7N7i_X6a81yQa8vtcxU06amK9pV9imWJnfHRSUcQ6),
+  Password: 7W!T*i74
+* [_Slides_](https://docs.google.com/presentation/d/16V82OIYfthb3fFlVoes9jZGKMgDIJZ55F8fXqd1M1hU/edit?usp=sharing)
+* [_Agenda_](https://etherpad.opendev.org/p/OSF_Community_Meeting_Q2)
+
+<br>
+
+The project update included:
+
+* The Alpha Milestone, [_completed in May_](https://www.airshipit.org/blog/airship-update-may-2020/)
+* Results of the [_Virtual PTG in June_](https://etherpad.opendev.org/p/airship-virtual-ptg-2020)
+* The newly introduced [_Vulnerability Management Process_](https://docs.airshipit.org/learn/vulnerabilities.html)
+* An update on GitHub Issues utilization
+* Notification of the Technical Committee (TC) and Working Committee (WC) elections
+* Announcement of the [_Airship User Survey_](https://www.surveymonkey.com/r/YKZ9NC2)
+
+<br>
+
+## **WORKING COMMITTEE ELECTION**
+
+The Technical Committee (TC) is excited to announce the 2020-2021 Working Committee (WC) election! This election will
```

Comment:
The usage of TC vs Technical Committee is a bit inconsistent.  We can introduce the TC abbreviation once, at its first introduction to the paper, and then we don't need to have the Technical Committee (TC) reusage.

Reply:
This is a nit I received from OSF, when I use just Technical Committee the merged post gets updated to include the abbreviation repeatedly.  See https://github.com/AirshipWeb/airship-website/pull/103 as an example